### PR TITLE
Committing the AllTeacher Tell Portal Automation Scripts

### PR DESCRIPTION
**Pull request by RameshAravindhQA**
Hi Satheesh,

Please review the code.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+			<version>4.13.0</version>
+		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/org.testng/testng -->
+		<dependency>
+			<groupId>org.testng</groupId>
+			<artifactId>testng</artifactId>
+			<version>7.4.0</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.apache.poi/poi-ooxml -->
```
Comment (satheesh12sk) [inline pom.xml]: This line is not needed; you can remove it, although it has been added twice.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+			<version>5.2.5</version>
+		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/io.cucumber/cucumber-java -->
+		<dependency>
+			<groupId>io.cucumber</groupId>
+			<artifactId>cucumber-java</artifactId>
+			<version>7.10.0</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/io.cucumber/cucumber-testng -->
```
Comment (satheesh12sk) [inline pom.xml]: Why should we use the latest dependencies for Cucumber testing? The latest is "7.18.1"

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+			<artifactId>cucumber-gherkin</artifactId>
+			<version>7.10.0</version>
+		</dependency>
+
+		<dependency>
+			<groupId>com.aventstack</groupId>
+			<artifactId>extentreports</artifactId>
+			<version>3.1.5</version>
+		</dependency>
+		 <dependency>
```
Comment (satheesh12sk) [inline pom.xml]: make this line aligned

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+		<!-- https://mvnrepository.com/artifact/io.cucumber/cucumber-java -->
+		<dependency>
+			<groupId>io.cucumber</groupId>
+			<artifactId>cucumber-java</artifactId>
+			<version>7.10.0</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/io.cucumber/cucumber-testng -->
+		<dependency>
+			<groupId>io.cucumber</groupId>
```
Comment (satheesh12sk) [inline pom.xml]: Why can't we use the latest dependencies for Cucumber TestNG? The latest is "7.18.1"

File: target/test-classes/Feature/Login.feature [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Feature: Validate the login functionality
+
+Scenario: Validate login with valid credentials
```
Comment (satheesh12sk) [inline target/test-classes/Feature/Login.feature:3]: For the scenario involving invalid credentials, we need to check both valid and invalid credentials

**Review by satheesh12sk — COMMENTED** [review]
Please check the comments and correct it

File: pom.xml (hunk 1 above)
Comment (RameshAravindhQA) [reply pom.xml]: Noted

File: pom.xml (hunk 3 above)
Comment (RameshAravindhQA) [reply pom.xml]: Noted

File: pom.xml (hunk 4 above)
Comment (RameshAravindhQA) [reply pom.xml]: Let me rectify it from the upcoming projects

File: target/test-classes/Feature/Login.feature (hunk 5 above)
Comment (RameshAravindhQA) [reply target/test-classes/Feature/Login.feature:3]: Noted. I refreshed the concepts that's why I didn't go for the multiple scenarios. Let me rectify it from upcoming works

File: pom.xml (hunk 1 above)
Comment (RameshAravindhQA) [reply pom.xml]: The Dependency needed for Apache poi two dependency to be added. There are two different dependencies

File: target/test-classes/Feature/Login.feature (hunk 5 above)
Comment (RameshAravindhQA) [reply target/test-classes/Feature/Login.feature:3]: As mentioned I refreshed the concepts that's why I didn't go for the multiple scenarios. Let me rectify it from upcoming works

**Review by RameshAravindhQA — COMMENTED** [review]
Hi Satheesh,

Please review the updated codes

**Review by RameshAravindhQA — COMMENTED** [review]
Updated

**Pull request merged**